modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/BasicInfoTab.tsx

```diff
@@ -84,19 +79,12 @@ export function BasicInfoTab({
 		Liferay.Language.Locale
 	>();
 
-	useEffect(() => {
-		const makeFetch = async () => {
-			if (objectDefinitionExternalReferenceCode) {
-				const objectDefinitionResponse = await API.getObjectDefinitionByExternalReferenceCode(
-					objectDefinitionExternalReferenceCode
-				);
-
-				setObjectDefinition(objectDefinitionResponse);
-			}
-		};
+	const isApproved = objectDefinition?.status!.label === 'approved';
 
-		makeFetch();
-	}, [objectDefinitionExternalReferenceCode]);
+	const [
+		objectDefinitionExternalReferenceCode2,
+		setObjectDefinitionExternalReferenceCode2,
+	] = useState<string>();
```

Comment:
Hey @igor-franca, Could you move this `useState` above the `isApproved` const? We must always keep the states declaration at the top of the file.